### PR TITLE
Fix/files/42460/breadcrumb conditional rendering

### LINKED ISSUE #42460: [BITV]: remove all native tooltips from breadcrumbs
I think it would be better to remove native tooltips from breadcrumbs (all `title` attrs) because then we will face with a problem again: tooltips are there but there is no other way to discover this information besides mouse over.

Screenshot from 2023-12-22 16-53-30

### PR DESCRIPTION
**Pull request by emoral435**
## ☑️ Resolves #42460

🖼️ Screenshots

🏡 After
firefox_7h4YBek9ip

🚧 Summary
Removed the title attribute from all breadcrumb headers from all but the first and last breadcrumbs - the first having an aria label, and the last being differently titled than the rest

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by JuliaKirschenheuter — DISMISSED** [review]
good but not completely right 

The last item should not have any `title`, it is just the same item as previous one

**Review by ShGKme — APPROVED** [review]

**Review by skjnldsv — CHANGES_REQUESTED** [review]
https://github.com/nextcloud/server/issues/42460#issuecomment-1874951161

**Comment by emoral435** [comment]
Will wait to see what we choose to do going forward before making any more changes :)

**Review by skjnldsv — APPROVED** [review]

File: apps/files/src/components/BreadCrumbs.vue [hunk 1]
```diff
@@ -30,7 +30,8 @@
 			v-bind="section"
 			dir="auto"
 			:to="section.to"
-			:title="titleForSection(section)"
+			:title="(index === 0 || index === sections.length - 1) ? titleForSection(section) : null"
+			:aria-description="(index === sections.length - 1) ? 'Reload current directory' : null"
```
Comment (ShGKme) [inline apps/files/src/components/BreadCrumbs.vue:34]: This string is not translated

**Review by ShGKme — CHANGES_REQUESTED** [review]

**Comment by emoral435** [comment]
Cleaned up the component a bit, I was making it a bit sloppier than it needed to be, to be honest!

**Review by ShGKme — APPROVED** [review]

**Comment by emoral435** [comment]
/ backport 255441980538bf46b7307a2aac257c4cd3c8a266 to stable28

**Pull request merged**